features/step_definitions/run_steps.rb

```diff
@@ -59,14 +59,9 @@
 
 
 Then(/^it runs the Git commands$/) do |expected_commands|
-  sha_regex = /\[SHA:(.+?)\]/
-
   # Replace SHA placeholders with the real SHAs
```

Comment:
comment can probably be removed now (or updated, it doesn't make sense as is anymore)